redirects-list.json

```diff
@@ -82,14 +82,16 @@
 
   "^/doc/vs-code-extension$                                                               https://marketplace.visualstudio.com/items?itemName=Iterative.dvc 302",
 
-  "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/projects/project-settings",
-  "^/doc/studio/user-guide/explore-experiments$                                           /doc/studio/user-guide/projects/explore-experiments",
-  "^/doc/studio/user-guide/visualize-experiments$                                         /doc/studio/user-guide/projects/visualize-experiments",
-  "^/doc/studio/user-guide/run-experiments$                                               /doc/studio/user-guide/projects/run-experiments",
-
-  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects/create-project",
-  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects/project-settings",
-  "^/doc/studio/user-guide/views/(.*)$                                                    /doc/studio/user-guide/projects/$1",
+  "^/doc/studio/view-settings$                                                            /doc/studio/user-guide/views/view-settings",
+  "^/doc/studio/user-guide/explore-experiments$                                           /doc/studio/user-guide/views/explore-experiments",
+  "^/doc/studio/user-guide/visualize-experiments$                                         /doc/studio/user-guide/views/visualize-experiments",
+  "^/doc/studio/user-guide/run-experiments$                                               /doc/studio/user-guide/views/run-experiments",
+  "^/doc/studio/user-guide/views/create-view$                                             /doc/studio/user-guide/projects/create-project 302",
+  "^/doc/studio/user-guide/views/view-settings$                                           /doc/studio/user-guide/projects/project-settings 302",
+  "^/doc/studio/user-guide/views/explore-experiments$                                     /doc/studio/user-guide/projects/explore-experiments 302",
+  "^/doc/studio/user-guide/views/visualize-experiments$                                   /doc/studio/user-guide/projects/visualize-experiments 302",
+  "^/doc/studio/user-guide/views/run-experiments$                                         /doc/studio/user-guide/projects/run-experiments 302",
+  "^/doc/studio/user-guide/views(/.*)$                                                    /doc/studio/user-guide/projects$1 302",
```

Comment:
Note that I'm making all the redirects to /doc/studio/user-guide/projects... 302s so we can just remove them in https://github.com/iterative/dvc.org/pull/3621, since they're pretty recent (hopefully not too many browsers have cached them as 301s).

Or should I leave them and add yet another redirect in #3621 ?